.rubocop.yml

```diff
@@ -8,11 +8,17 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 50
```

Comment:
In line 11, you change the max:20 to max:50, keep the original file. You can disable a particular rubocop. 
In this case use: 
Put this line at the beginning  of the code 
`# rubocop:disable Metrics/MethodLength`
Put this line at the end of the code 
`# rubocop:enable Metrics/MethodLength`